rust/crd/src/lib.rs

```diff
@@ -387,12 +425,19 @@ impl Configuration for HbaseConfigFragment {
         match file {
             HBASE_ENV_SH => {
                 result.insert(HBASE_MANAGES_ZK.to_string(), Some("false".to_string()));
-                let mut all_hbase_opts = format!("-Djava.security.properties={CONFIG_DIR_NAME}/{JVM_SECURITY_PROPERTIES_FILE} -javaagent:/stackable/jmx/jmx_prometheus_javaagent.jar={METRICS_PORT}:/stackable/jmx/{role_name}.yaml");
+
+                // As we don't have access to the clusterConfig, we always enable the `-Djava.security.krb5.conf`
+                // config, besides it is not always being used.
```

Comment:
That has no other implications? Always providing the krb5.conf?
```suggestion
                // As we don't have access to the clusterConfig, we always enable the `-Djava.security.krb5.conf`
                // config, besides it not always being used.
```

Reply:
Not that I'm aware of, all tests pass